crates/diesel_models/src/payment_intent.rs

```diff
@@ -233,18 +232,15 @@ pub struct PaymentIntentUpdateInternal {
     #[diesel(deserialize_as = super::OptionalDieselArray<pii::SecretSerdeValue>)]
     pub order_details: Option<Vec<pii::SecretSerdeValue>>,
     pub attempt_count: Option<i16>,
-    pub profile_id: Option<String>,
```

Comment:
why profile_id got removed?

Reply:
we should not be able to update `profile_id` in payment intent.